tests/integration/account/test_account.py

```diff
@@ -0,0 +1,253 @@
+from tests.integration.helpers import exec_test_command
+
+BASE_CMD = ["linode-cli", "account"]
+
+
+def test_account_transfer():
+    res = (
+        exec_test_command(BASE_CMD + ["transfer", "--text", "--delimiter=,"])
+        .stdout.decode()
+        .rstrip()
+    )
+    lines = res.splitlines()
+
+    headers = ["billable", "quota", "used"]
+    for header in headers:
+        assert header in lines[0]
+
+
+def test_region_availability():
+    res = (
+        exec_test_command(
+            BASE_CMD
+            + ["get-account-availability", "us-east", "--text", "--delimiter=,"]
+        )
+        .stdout.decode()
+        .rstrip()
+    )
+    lines = res.splitlines()
+
+    headers = ["region", "unavailable"]
```

Comment:
Just a general comment here, if we are only comparing headers for all the tests might be cleaner to create one helper function that can be called when asserting